tarmac/src/main/scala/nl/lumc/sasc/biopet/pipelines/tarmac/Tarmac.scala

```diff
@@ -29,13 +38,90 @@ class Tarmac(val parent: Configurable) extends QScript with PedigreeQscript with
 
   class Sample(name: String) extends AbstractSample(name) {
 
-    val inputCountFile: Option[File] = config("count_file")
+    val inputXhmmCountFile: Option[File] = config("xhmm_count_file")
+    val inputWisecondorCountFile: Option[File] = config("wisecondor_count_file")
     val bamFile: Option[File] = config("bam")
 
+    /**
+     * Create XHMM count file or create link to input count file
+     * Precedence is given to existing count files.
+     * Returns a disjunction where right is the file, and left is
+     * a potential error message
+     */
+    protected lazy val outputXhmmCountJob: String \/ QFunction = {
+      val outFile = new File(sampleDir + File.separator + s"$name.dcov")
+      (inputXhmmCountFile, bamFile) match {
+        case (Some(f), _) => {
+          val ln = new Ln(root)
```

Comment:
Maybe add here something like this:
`if (bamFile.isDefined) logger.warn(s"bam file and count file are both given for $sampleId, bam file is ignored")`